tests/bluetooth/common/testlib/include/bs_helpers/testcase.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "bs_tracing.h"
+#include "bs_types.h"
+#include "bstests.h"
+
+extern enum bst_result_t bst_result;
+
+#define TEST_START(timeout_us, message)                                                            \
+	do {                                                                                       \
+		bst_ticker_set_next_tick_absolute((bs_time_t)timeout_us);                          \
+		bst_result = In_progress;                                                          \
+		bs_trace_info_time(1, "Test start: %s (timeout at %u us)\n", message, timeout_us); \
```

Comment:
```suggestion
		bs_trace_info_time(1, "Test start: %s (timeout at %llu us)\n", message,            \
				   timeout_us);                                                    \
```

Reply:
Actually, please use `%"PRItime"` for bs_time_t.